ui-v2/app/styles/components/app-view/layout.scss

```diff
@@ -10,6 +10,15 @@
   display: flex;
   align-items: flex-start;
 }
+%app-view header dl {
+  float: left;
+  margin-top: 25px;
+  margin-right: 50px;
+  margin-bottom: 20px;
+}
+%app-view header dt {
+  font-weight: bold;
+}
```

Comment:
This controls the rendering here:

<img width="765" alt="screenshot 2019-02-08 at 15 25 07" src="https://user-images.githubusercontent.com/554604/52487561-d25e9680-2bb5-11e9-87c6-31a0214cd88a.png">